yolapy/models/site.py

```diff
@@ -14,4 +14,12 @@ def __init__(self, **kwargs):
     def get(cls, site_id):
         """Get a site from the Yola API."""
         site_attributes = Yola().get_site(site_id)
-        return Site(**site_attributes)
+        return cls(**site_attributes)
+
+    @classmethod
+    def for_user(cls, user_id):
```

Comment:
I think `Site.get(...)` will raise an exception if the site doesn't exist and this will return `None`. I'm not sure that inconsistency makes sense. It's a point worth considering at the very least.

I likely would have changed `get()` to accept either a `site_id` or a `user_id` and masked the not found error states in a custom exception.